src/google/Scatter.js

```diff
@@ -0,0 +1,215 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "./CommonND", "../common/HTMLWidget"], factory);
+    } else {
+        root.Scatter = factory(root.d3, root.google_CommonND, root.common_HTMLWidget);
+    }
+}(this, function (d3, CommonND, HTMLWidget) {
+
+    function Scatter() {
+        CommonND.call(this);
+        this._class = "google_Scatter";
```

Comment:
The _class should be moved into the prototype now (this is a recent-ish change) see https://github.com/hpcc-systems/Visualization/blob/master/src/google/Column.js#L16